### DISCUSSION TITLE
Play as Nemesis?

### DISCUSSION THREAD

**Discussion by RyuuzakiL5** [IntelOrca/biorand#440] [Q&A] [answered]
I watched Bawkbasoup live where he play as Nemesis in Resident Evil 2, he used the 3.0.3 version, but I couldn't find it the option to let me play as him and his randomized voice. Also, there is no folder with his name. Is that a special version or to play as him I need to choose a character not named Nemesis?

https://youtu.be/RRpV5ucKROk?t=3216

**Comment by IntelOrca** [comment]
It is available as a separate add-on:
https://www.moddb.com/games/resident-evil-2/downloads/re2-dodging-skin-playable-boss-skin-for-biorand

**Reply by RyuuzakiL5** [reply]
Woooow! Thank you!

**Comment by IntelOrca** [comment, marked as answer]
Now available officially in biorand for both RE 2 and RE 3. RE 1 will come soon.
